src/firefly/js/ui/CatalogSearchMethodType.jsx

```diff
@@ -47,16 +52,33 @@ export class CatalogSearchMethodType extends Component {
         });
     }
 
+    componentWillReceiveProps(nextProps) {
+        // const fields= FieldGroupUtils.getGroupFields(this.nextProps.groupKey);
+        const {coneMax, boxMax, groupKey}= nextProps;
+        if (coneMax && groupKey) {
```

Comment:
I am not sure if I understand the logic. should the case 'boxMax&&groupKey' be considered?